implants/pkg/tavern/graphql/mutations.graphql

```diff
@@ -0,0 +1,34 @@
+mutation ClaimTasks($input: ClaimTasksInput!) {
```

Comment:
Thoughts on reusing the existing schema instead of copying a special one down to imix?

Reply:
Ah I see it in go generate